From c6ed030f22ef9632b6323389c982a672ae0451b6 Mon Sep 17 00:00:00 2001
From: Simon Kitching <skitching@apache.org>
Date: Mon, 9 Jun 2008 15:01:34 +0000
Subject: [PATCH] Fix DIGESTER-125; clear inputSources list in method
 Digester.clear(). Thanks to Andrey Polyakov for pointing this out.

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/digester/trunk@665710 13f79535-47bb-0310-9956-ffa450edef68
---
 src/java/org/apache/commons/digester/Digester.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/java/org/apache/commons/digester/Digester.java b/src/java/org/apache/commons/digester/Digester.java
index 8ecb0552c..b98f64f13 100644
--- a/src/java/org/apache/commons/digester/Digester.java
+++ b/src/java/org/apache/commons/digester/Digester.java
@@ -2988,6 +2988,7 @@ protected void cleanup() {
                 ; // Fall through so we get them all
             }
         }
+        inputSources.clear();
 
     }
 
